ArduPlane/servos.cpp

```diff
@@ -253,7 +253,10 @@ void Plane::dspoiler_update(void)
         }
     }
 
-    const int16_t weight_outer = g2.crow_flap_weight_outer.get();
+    int16_t weight_outer = g2.crow_flap_weight_outer.get();
+    if (crow_mode == Plane::Crow_Mode::CROW_DISABLED) {   //override totally aileron crow if crow RC switch set to disabled
```

Comment:
Isn't this comment misleading in that crow_mode can take a value based on a parameter?

Also, this doesn't match the PR title.

Reply:
CROW operation has two modes: Progressive and not progressive (full range?). This PR is about letting a user decide whether enable CROW at all, or force Progressive CROW (whether or not set up in params), or just let the existing param setup work...
use cases: he sets it up in params...then the type of CROW cant be changed on approach if needed or he cant disable it to use just FLAPs (no CROW)....now he can do anything he wishes including no using CROW, using it as Progressive or using it as full CROW